net/ifstat/Makefile

```diff
@@ -36,6 +36,16 @@ define Package/ifstat/description
 	interfaces using SNMP.
 endef
 
+define Package/ifstat/config
+	source "$(SOURCE)/Config.in"
+endef
+
+ifeq ($(CONFIG_IFSTAT_SNMP),y)
```

Comment:
This can be replaced with CONFIGURE_ARGS += --with$(if $(CONFIG_IFSTAT_SNMP),,out)-snmp

Reply:
Nifty. Updated.